ember-bootstrap/addon/components/bs-carousel.hbs

```diff
@@ -1,47 +1,66 @@
-<div
-  {{!-- template-lint-disable no-positive-tabindex --}}
-  tabindex={{this.tabindex}}
-  class="carousel slide {{if this.carouselFade "carousel-fade"}}"
-  ...attributes
-  {{on "keydown" this.handleKeyDown}}
-  {{on "mouseenter" this.handleMouseEnter}}
-  {{on "mouseleave" this.handleMouseLeave}}
-  {{did-update this.childSlidesObserver this.childSlides this.autoPlay}}
-  {{did-update this.indexObserver this.index}}
->
-  {{#if this.showIndicators}}
-    <ol class="carousel-indicators">
-      {{#each this.indicators as |indicator _index|}}
-        <li class={{if (bs-eq this.currentIndex _index) "active"}} {{on "click" (fn this.toSlide _index)}} role="button"></li>
-      {{/each}}
-    </ol>
-  {{/if}}
-
-  <div class="carousel-inner" role="listbox">
-    {{yield
-      (hash
-        slide=(component (ensure-safe-component (bs-default @slideComponent (component "bs-carousel/slide")))
-          currentSlide=this.currentSlide
-          directionalClassName=this.directionalClassName
-          followingSlide=this.followingSlide
-          orderClassName=this.orderClassName
-          presentationState=this.presentationState
-          registerChild=this.registerChild
-          unregisterChild=this.unregisterChild
-        )
-      )
-    }}
-  </div>
-
-  {{#if this.showControls}}
-    <a class="carousel-control-prev" href="#{{this.elementId}}" {{on "click" this.toPrevSlide}} role="button">
-      <span aria-hidden="true" class="carousel-control-prev-icon"></span>
-      <span class={{if (macroCondition (macroGetOwnConfig "isBS5")) "visually-hidden" "sr-only"}}>{{this.prevControlLabel}}</span>
-    </a>
-    <a class="carousel-control-next" href="#{{this.elementId}}" {{on "click" this.toNextSlide}} role="button">
-      <span aria-hidden="true" class="carousel-control-next-icon"></span>
-      <span class={{if (macroCondition (macroGetOwnConfig "isBS5")) "visually-hidden" "sr-only"}}>{{this.nextControlLabel}}</span>
-    </a>
-  {{/if}}
-
+<div
+  {{!-- template-lint-disable no-positive-tabindex --}}
+  tabindex={{this.tabindex}}
+  class="carousel slide {{if this.carouselFade "carousel-fade"}}"
+  ...attributes
+  {{on "keydown" this.handleKeyDown}}
+  {{on "mouseenter" this.handleMouseEnter}}
+  {{on "mouseleave" this.handleMouseLeave}}
+  {{did-update this.childSlidesObserver this.childSlides this.autoPlay}}
+  {{did-update this.indexObserver this.index}}
+>
+  {{#if this.showIndicators}}
+    {{#if (macroCondition (macroGetOwnConfig "isBS5"))}}
+      <div class="carousel-indicators">
+        {{#each this.indicators as |indicator _index|}}
+          <button
+            data-bs-target
```

Comment:
What is this data attribute used for?

Reply:
It's required for the indicator CSS to work.
This is the relevant Boostrap 5 stylesheet: https://github.com/twbs/bootstrap/blob/11272eca59e426906f1ee105371fe8477266c24b/scss/_carousel.scss#L169